code/game/objects/items/legcuffs.dm

```diff
@@ -92,3 +93,49 @@
 					var/mob/living/simple_animal/SA = AM
 					SA.health -= 20
 	..()
+
+
+/obj/item/restraint/legcuffs/xeno_restraints
+	name = "xeno restraints"
+	desc = "Use this to hold xenomorphic creatures moderately-safely."
+	gender = PLURAL
+	flags_atom = FPRINT|CONDUCT
+	flags_equip_slot = SLOT_WAIST
+	throwforce = 5
+	w_class = SIZE_SMALL
+	throw_speed = SPEED_SLOW
+	throw_range = 5
+	matter = list("metal" = 500)
+
+	breakouttime = 2 MINUTES
+
+/obj/item/restraint/legcuffs/xeno_restraints/apply_legcuffs(mob/living/carbon/target, mob/user)
+	return FALSE
+
+/obj/item/restraint/legcuffs/xeno_restraints/attack(mob/living/carbon/C as mob, mob/user as mob)
+	if(!istype(C, /mob/living/carbon/xenomorph))
+		to_chat(user, SPAN_DANGER("The cuffs do not fit!"))
+		return
+	if(!C.legcuffed)
+		var/turf/p_loc = user.loc
+		var/turf/p_loc_m = C.loc
```

Comment:
these var names aren't very descriptive and I can't really figure out what they're meant to represent